scripts/travis/deploy-linux-build.sh

```diff
@@ -64,19 +64,21 @@ echo "Deploy travis build on the server..."
 # Add server public key to known hosts.
 echo $DEPLOY_SSH_KEY >> $HOME/.ssh/known_hosts
 
-# Send build to server.
+# Remove previous build from server.
```

Comment:
Don't you want to preserve the previous build?

Reply:
no. The test runner copy the build somewhere else so it won't affect the ruuning test job or something